tests/functional/s/syntax/syntax_error_invalid_encoding.txt

```diff
@@ -0,0 +1 @@
+syntax-error:1:0:None:None::"Parsing failed: 'unknown encoding for '/Users/daniel/DocumentenLaptop/Programming/Github/pylint/tests/functional/s/syntax/syntax_error_invalid_encoding.py': lala'":HIGH
```

Comment:
```suggestion
syntax-error:1:0:None:None::"Parsing failed: 'unknown encoding for '/Users/daniel/DocumentenLaptop/Programming/Github/pylint/tests/functional/s/syntax/syntax_error_invalid_encoding.py': lala'":HIGH
```
This can't work, we probably need a unit test for this or to change to a relative path.

Reply:
Oh yeah. I'll make a unittest